pygmt/src/grdcontour.py

```diff
@@ -95,6 +97,33 @@ def grdcontour(self, grid, **kwargs):
         separator for the two labels instead.
     {perspective}
     {transparency}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 15 arc minute grid with "gridline" registration in a specified region
```

Comment:
Fix the failing "Style Checks".
```suggestion
    >>> # load the 15 arc minute grid with "gridline" registration 
    >>> # in a specified region
```